integration/tests/basic/helpers/rpc_checks.py

```diff
@@ -103,16 +103,16 @@ def assert_log_field_in_neon_trx_receipt(response, events_count):
     assert event_types == expected_event_types, f"Actual: {event_types}; Expected: {expected_event_types}"
 
 
-def assert_fields_are_hex(object, expected_hex_fields):
-    if isinstance(object, SimpleNamespace):
+def assert_fields_are_hex(_object, expected_hex_fields):
```

Comment:
why you think _object is better than object? In general "_" use for internal variables, in this case I think "object" is better